### DISCUSSION TITLE
tailwind / headless ui - how to setup icon center in popover?

### DISCUSSION THREAD

**Discussion by mxcdh** [tailwindlabs/tailwindcss#9097] [Help]
I have problem with changed icon in this component:
https://headlessui.com/react/popover

When I setup custom icons from herocions, my icons look like:
- a - orginal 
- b - my icons

[![screenshot][1]][1]

  [1]: https://i.stack.imgur.com/3kQaI.png

My code is bellow:
https://codesandbox.io/p/github/mxcdh/codesandbox-react-tailwind-geadless-ui-popover/draft/vigilant-meninsky?create=true&import=true&file=%2Fsrc%2Fcomponents%2FHeader.tsx

What I should to do? And when is bug?

**Comment by wongjn** [comment]
The rounded pale rectangle and the icon inside it is inside an SVG. For the first icon, the glyph part of the SVG is visually aligned to the center of the SVG, but in the second icon, the glyph is aligned near to the top-left corner:

```svg
<path
  d="M7 8h10M7 12h4m1 8l-4-4H5a2 2 0 01-2-2V6a2 2 0 012-2h14a2 2 0 012 2v8a2 2 0 01-2 2h-3l-4 4z"
  stroke="#FB923C"
  strokeWidth="2"
/>
```
You need to edit the `IconTwo` SVG so that this `path` element glyph is repositioned to the center.
